.travis.yml

```diff
@@ -12,15 +12,19 @@ stages:
   - name: unsupported
     if: branch = master AND type = push
 
+.disable_global: &disable_global
```

Comment:
note this convention is being used in lots of coala repos now
https://github.com/coala/coala-bears/blob/master/.travis.yml#L9